src/components/Home/styled.js

```diff
@@ -1,40 +1,76 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 export const App = styled.div`
   text-align: center;
+  height: 100vh;
+  ${({ background }) =>
+    background &&
+    css`
+      background-image: url(\"data:image/svg+xml;utf8,${background}\");
+      background-size: cover;
+      min-height: 100%;
+    `}
 `;
 
 export const AppHeader = styled.header`
-  background-color: #282c34;
-  min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
+  background: rgba(255,255,255,0.3);
+  backdrop-filter: blur(4px);
+  height: 100%;
+  & .title {
+    display: block;
+    width: 100%;
+    background: var(--blue);
+    color: #fff;
+    font-size: calc(10px + 15vmin);
```

Comment:
I would use `em` or `rem` for all of the font-sizes and make sure the elements work relative to each other. If we want to add scaling, I think it would be better to do it globally. What do you think?